spec/integration/report_spec.rb

```diff
@@ -4,7 +4,7 @@
   before :each do
     webauth = double('WebAuth', :login => 'sunetid', :attributes => {'DISPLAYNAME' => 'Example User'}, :privgroup => ADMIN_GROUPS.first)
     @current_user = User.find_or_create_by_webauth(webauth)
-    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@current_user)
+    allow_any_instance_of(CatalogController).to receive(:current_user).and_return(@current_user)
```

Comment:
Yeah.. from rspec-mocks documentation:
> - Using this feature is often a design smell. It may be that your test is trying to do too much or that the object under test is too complex.
> - It is the most complicated feature of rspec-mocks, and has historically received the most bug reports. (None of the core team actively use it, which doesn't help.)
http://www.relishapp.com/rspec/rspec-mocks/v/3-4/docs/working-with-legacy-code/any-instance